gbus/bus.go

```diff
@@ -75,7 +75,9 @@ var (
 	//for a random retry time. Default is 10 but it is configurable.
 	BaseRetryDuration = 10 * time.Millisecond
 	//RPCHeaderName used to define the header in grabbit for RPC
-	RPCHeaderName = "x-grabbit-msg-rpc-id"
+	RPCHeaderName         = "x-grabbit-msg-rpc-id"
```

Comment:
shouldn't they all be consts?

Reply:
I agree, I just wasn't sure if we want to do it in this PR as it would cause it to change a lot of code which is not related to what I am doing here